packages/config/src/configDefaults.ts

```diff
@@ -158,7 +153,7 @@ export const configProps = ({
           config = {};
         }
 
-        config = assignIn({}, defaultTXValues, config);
+        config = assignIn({}, config);
```

Comment:
Do we even need this `assignIn` now that we're not really merging anything?

Reply:
hm. I guess we don't actually need this line anymore. I can remove it.